exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/project/ProjectMemoryManager.java

```diff
@@ -267,8 +284,13 @@ public void update() {
                 OutputWidthExpression savedWidthExpr = columnWidthInfo.getOutputExpression();
                 OutputWidthVisitorState state = new OutputWidthVisitorState(this);
                 OutputWidthExpression reducedExpr = savedWidthExpr.accept(new OutputWidthVisitor(), state);
-                width = ((FixedLenExpr)reducedExpr).getWidth();
+                width = ((FixedLenExpr)reducedExpr).getDataWidth();
                 Preconditions.checkState(width >= 0);
+                int metadataWidth = getMetadataWidth(columnWidthInfo.outputVV);
+                Preconditions.checkState(metadataWidth >= 0);
```

Comment:
This check is an overkill; seems that all the computations in getMetadataWidth() are bound to return >= 0 values.

Reply:
Will remove.